pkg/openid4vp/openid4vp.go

```diff
@@ -148,8 +148,31 @@ func (o *Interaction) VerifierDisplayData() (*VerifierDisplayData, error) {
 	}, nil
 }
 
+type presentOpts struct {
+	ignoreConstraints bool
+}
+
 // PresentCredential presents credentials to redirect uri from request object.
 func (o *Interaction) PresentCredential(credentials []*verifiable.Credential) error {
+	return o.presentCredentials(
+		credentials,
+		&presentOpts{},
+	)
+}
+
+// PresentCredentialUnsafe presents credentials to redirect uri from request object.
+// If one credential is provided, this will skip presentation definition constraint validation.
```

Comment:
Just to clarify: do you mean that the presentation definition constraint validation is only skipped if one (and only one) credential is provided? I noticed that the Gomobile version's comment doesn't state this.

Reply:
Good point, I'll clarify.

Reply:
Actually, I should also add a check that fails if more than one VC is passed into unsafe, since that shouldn't work.

Reply:
You can just make it accept only one credential instead of an array.

Reply:
Done